<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordWindow</class>
 <widget class="QDialog" name="PasswordWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>129</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change password</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="passwordFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="innerHorizontalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="labelVerticalLayout">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="QLabel" name="oldPasswordLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&amp;Old password</string>
          </property>
          <property name="buddy">
           <cstring>oldPasswordLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="passwordLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&amp;New password</string>
          </property>
          <property name="buddy">
           <cstring>passwordLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="confirmPasswordLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&amp;Confirm new password</string>
          </property>
          <property name="buddy">
           <cstring>confirmPasswordLineEdit</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="inputVerticalLayout">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="QLineEdit" name="oldPasswordLineEdit">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passwordLineEdit">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="confirmPasswordLineEdit">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dialogButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>oldPasswordLineEdit</tabstop>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>confirmPasswordLineEdit</tabstop>
  <tabstop>dialogButtonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
